README.md

```diff
@@ -102,45 +102,54 @@ Make a copy of your access key ID and secret access key. You'll need to include
     npm run db:migrate -- up
     ```
 
-13. Run the server:
+13. Seed your development database with test member and project data:
+    ```bash
+    npm run db:copy -- <STATE>
```

Comment:
The state option doesn't apply for `idm`. My fault for not noting that at in the request.